### PR TITLE
Create makefile for automated setup and testing environment using docker

### PR DESCRIPTION
**Pull request by JorgeMartinezG**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+list:
+	docker-compose ps
+
+tests:test-client test-server
+
+test-client:
+	docker-compose exec app sh -c "cd /usr/src/app/tests/client && ../../client/node_modules/.bin/karma start ./karma.conf.js --single-run --browsers PhantomJS"
+
+test-server:
+	docker-compose exec app sh -c "python -m unittest tests/server"
```
Comment (xamanu) [inline Makefile]: Missing `discover` for `docker-compose exec app sh -c "python -m unittest discover tests/server"`

Comment (JorgeMartinezG) [reply Makefile]: Check

**Comment by JorgeMartinezG** [comment]
@xamanu applied changes and suggestions.

File: README.md [hunk 2]
```diff
@@ -171,6 +171,35 @@ If you plan to only work on the API you only have to build the server architectu
 * Point your browser to:
     * [http://localhost:5000/api-docs](http://localhost:5000/api-docs)
 
+## Docker setup and testing for (non)developers
```
Comment (xamanu) [inline README.md]: I suggest to move this into a separate file. Probably it makes sense to have one for the whole docker setup. I expect there to be more to be documented.

Comment (JorgeMartinezG) [reply README.md]: Check!. I agree. We would need to include commands such as database migration, coverage among others

**Comment by JorgeMartinezG** [comment]
@xamanu This branch was rebased to be one commit ahead of `develop`

**Comment by xamanu** [comment]
Thanks @JorgeMartinezG!

**Pull request merged**